t/89async-method-check.t

```diff
@@ -43,7 +43,6 @@ my %dbh_args = (
     parse_trace_flag    => ['SQL'],
     parse_trace_flags   => ['SQL'],
     trace_msg           => ['message'],
-    visit_child_handles => [sub { }],
```

Comment:
Can't we check whether it's available and test it if it is?

Reply:
Seems yes, `DBI->can` is working.